test/regress/CMakeLists.txt

```diff
@@ -2548,6 +2547,8 @@ set(regression_disabled_tests
   ###
   regress1/bug472.smt2
   regress1/datatypes/non-simple-rec-set.smt2
+  # TODO: fix models (projects #276)
```

Comment:
Can't find this project:
https://github.com/orgs/CVC4/projects/276

Just to make sure -- is there a specific issue that corresponds to this TODO? If not, can you create one in the corresponding project?

see guideline here: https://github.com/CVC4/CVC4/wiki/Code-Style-Guidelines#todofixme-comments

Reply:
https://github.com/CVC4/cvc4-projects/issues/276